understanding/content-on-hover-or-focus.html

```diff
@@ -55,348 +53,279 @@
-                     viewport may trigger the additional content, making it difficult for a user to pan
-                     without re-triggering the content. A keyboard means of dismissing the additional content
-                     provides a workaround.
+                  <p>この条件の意図は、追加のコンテンツがページの元のコンテンツを確認する又は操作することを妨げないようにすることである。拡大時、ビューポート内で表示されるページの部分はかなり縮小する。マウスの利用者は、拡大したビューポートをパンして画面の他の部分を表示するために、頻繁にポインタを動かす。しかし、この限られたビューポートで表示されるページのほぼ全域が追加のコンテンツをトリガーするかもしれず、利用者がコンテンツを再トリガーせずにパンすることを困難にする。キーボードで追加コンテンツを非表示にする方法は、回避方法を提供する。
                   </p>
                   			
-                  <p>Alternatively, low vision users who can only navigate via the keyboard do not want
-                     the small area of their magnified viewport cluttered with hover text. They need a
-                     keyboard method of dismissing something that is obscuring the current focal area.
+                  <p>代わりに、キーボードを用いてのみナビゲートするロービジョンの利用者は、拡大されたビューポートの小さい領域がホバーテキストで雑然としてほしくない。彼らは現在のフォーカスが当たっている領域を覆い隠すものをキーボードで非表示にする方法が必要である。
                   </p>
                   		
-                  <p>Two methods may be used to satisfy this condition and prevent such interference:</p>
+                  <p>この条件を満たし、このような干渉を防ぐために、二つの方法が使用できる:</p>
                   			
                   <ol>
                      			
-                     <li>Position the additional content so that it does not obscure any other content including
-                        the trigger, with the exception of white space and purely decorative content, such
-                        as a background graphic which provides no information.
+                     <li>余白及び情報を提供しない背景画像などの純粋な装飾コンテンツを除き、トリガーを含む他のあらゆるコンテンツを隠さないよう追加のコンテンツを配置する。
                      </li>
                      			
-                     <li>Provide a mechanism to easily dismiss the additional content, such as by pressing
-                        Escape.
+                     <li>Esc キーを押すなど、追加のコンテンツを容易に非表示にするメカニズムを提供する。
                      </li>
                      			
                   </ol>
                   			
-                  <p>For most triggers of relatively small size, it is desirable for both methods to be
-                     implemented.  If the trigger is large, noticing the additional content may be of concern
-                     if it appears away from the trigger.  In those cases, only the second method may be
-                     appropriate.
+                  <p>ほとんどの比較的小さいサイズのトリガーには、両方の方法が実装されることが望ましい。トリガーが大きい場合、追加コンテンツがトリガーより遠くに現れると、気づくことが難しいかもしれない。そのような場合は、二つ目の方法のみが適切かもしれない。
+
+
                   </p>
                   			
-                  <p>The success criterion allows for input error messages to persist as there are cases
-                     that require attention, explicit confirmation or remedial action.
+                  <p>注目、明示的な確認、又は是正措置が必要な場合があるため、達成基準により、入力エラーメッセージを持続させることができる。
```

Comment:
これちょっとわかりにくいですね。こんな感じでしょうか。The success criterion allows... を全般的にどうするか、という話のような気もしますが。

```suggestion
                  <p>注目、明示的な確認、又は是正措置が必要な場合があるため、達成基準は、入力エラーメッセージを持続させることを許している。
```